src/main/java/uk/gov/register/providers/FieldConversionExceptionMapper.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.register.providers;
+
+import uk.gov.register.exceptions.FieldConversionException;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+@Provider
+public class FieldConversionExceptionMapper implements ExceptionMapper<FieldConversionException> {
+
+    @Override
+    public Response toResponse(final FieldConversionException exception) {
+        return Response.status(500).entity("[ERROR] " + exception.getMessage()).build();
```

Comment:
Can we use the `ExceptionView` to display this error? Look at the `ThrowableExceptionMapper` as an example, where it uses `.entity(viewFactory.exceptionServerErrorView())`. We should also do this for "register undefined" but we will do that separately.